tools/src/bin/tquic_client.rs

```diff
@@ -228,6 +232,14 @@ pub struct ClientOpt {
     /// Batch size for sending packets.
     #[clap(long, default_value = "1", value_name = "NUM")]
     pub send_batch_size: usize,
+
+    /// Congestion Control Algorithm.
+    #[clap(short = 'C', long, default_value = "cubic")]
+    pub cong_control: CongestionControlAlgorithm,
```

Comment:
Why would this still exist? As we already have an option for congestion_control_algor.